presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInstaller.java

```diff
@@ -85,21 +93,56 @@ private static void installUsingRpm(String pathToRpm, boolean checkDependencies)
         if (installRpm != 0) {
             throw new PrestoManagerException("Failed to install Presto", installRpm);
         }
-        addTpchCatalog();
+        addConfigFiles(CONFIG_DIR, DEFAULT_CONFIG_DIR);
+        addConnectors(CATALOG_DIR, DEFAULT_CATALOG_DIR);
         LOGGER.debug("Successfully installed Presto");
     }
 
-    private static void addTpchCatalog()
+    private static void addTpchCatalog(Path catalogDir)
             throws PrestoManagerException
     {
         try {
-            if (!isDirectory(CATALOG_DIR)) {
-                createDirectory(CATALOG_DIR);
-                write(CATALOG_DIR.resolve("tpch.properties"), "connector.name=tpch".getBytes());
+            if (!isDirectory(catalogDir)) {
+                createDir(catalogDir);
+                write(catalogDir.resolve("tpch.properties"), "connector.name=tpch".getBytes());
             }
         }
-        catch (IOException e) {
+        catch (IOException | PrestoManagerException e) {
             throw new PrestoManagerException("Failed to add tpch catalog", e);
         }
     }
+
+    private static void addConfigFiles(Path configDir, Path defaultConfigDir)
```

Comment:
I would name this something like `useDefaultConfig`, and it should not take the default configuration as an argument; it should reference the constant directly.

I would also probably have another function called `addDefaultConfig` that does the same thing, but doesn't delete the existing files first. (Call that one here.)